### PR TITLE
Add Returns section to the docstring

### LINKED ISSUE #7797: griddata docstring is missing the "Returns" section.
Like the title says, the docstring for `scipy.interpolate.griddata` doesn't have a "Returns" section.

### PR DESCRIPTION
**Pull request by souravsingh**
Fixes #7797

### CODE REVIEW

**Comment by souravsingh** [comment]
@WarrenWeckesser Can you please check if the info is written correctly?

File: scipy/interpolate/ndgriddata.py [hunk 1]
```diff
@@ -132,6 +132,13 @@ def griddata(points, values, xi, method='linear', fill_value=np.nan,
         incommensurable units and differ by many orders of magnitude.
 
         .. versionadded:: 0.14.0
+        
+    Returns
+    -------
+
```
Comment (WarrenWeckesser) [inline scipy/interpolate/ndgriddata.py]: Remove this blank line.

File: scipy/interpolate/ndgriddata.py [hunk 2]
```diff
@@ -132,6 +132,13 @@ def griddata(points, values, xi, method='linear', fill_value=np.nan,
         incommensurable units and differ by many orders of magnitude.
 
         .. versionadded:: 0.14.0
+        
+    Returns
+    -------
+    
+    ip: ndarray
```
Comment (WarrenWeckesser) [inline scipy/interpolate/ndgriddata.py]: There is no need to create a name for the return value; this line can be simply `ndarray`.  See the subsection "5. Returns" in the file https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections

File: scipy/interpolate/ndgriddata.py [hunk 3]
```diff
@@ -132,6 +132,13 @@ def griddata(points, values, xi, method='linear', fill_value=np.nan,
...
+        Interpolation points of dimensions xi and based on the method
```
Comment (WarrenWeckesser) [inline scipy/interpolate/ndgriddata.py]: How about something like "Array with the same shape as `xi` containing the interpolated values."?

**Review by WarrenWeckesser — DISMISSED** [review]

File: scipy/interpolate/ndgriddata.py (hunk 3 above)
Comment (WarrenWeckesser) [reply scipy/interpolate/ndgriddata.py]: Wait, ignore that comment.  The returned array is not "the same shape as `xi`".

Comment (WarrenWeckesser) [reply scipy/interpolate/ndgriddata.py]: It turns out the description of `xi` in the "Parameters" section is misleading.  It says `"xi : 2-D ndarray of float or tuple of 1-D array, shape (M, D)"`, but if `xi` is a tuple, the values in the tuple do *not* have to be 1-D.  Just look at the example, where `grid_x` and `grid_y` are 2-D.  In fact, the arrays in the tuple can have arbitrary shape, as long as the shapes are compatible for broadcasting.  And if `xi` is a numpy array, it does not have to be 2-D.  Apparently just the *last* dimension of `xi` must be `D`.

**Comment by WarrenWeckesser** [comment]
@souravsingh It turns out that the description of `xi` in the "Parameters" section also needs to be fixed.  If you would like to work on that, go ahead, but expected several iterations of edits.  Or, you could just fix the "Returns" section to something like

    Returns
    -------
    ndarray
        Array of interpolated values.

and leave fixing the description of `xi` to a separate pull request.

**Comment by souravsingh** [comment]
@WarrenWeckesser I have made the fixes.

**Comment by souravsingh** [comment]
Is the patch ready for merging?

**Comment by WarrenWeckesser** [comment]
Merged, thanks @souravsingh.

**Pull request merged**